app/pages/sections/catalogue-solutions/catalogue-solutions/ui-tests/general.ui.test.js

```diff
@@ -131,6 +131,17 @@ test('should render the Add Catalogue Solution button', async (t) => {
     .expect(await extractInnerText(addSolutionButton)).eql(content.addSolutionButtonText);
 });
 
+test('should navigate to /organisation/order-id/catalogue-solutions/select with clicking the Add Catalogue Solution button', async (t) => {
```

Comment:
should navigate to /organisation/order-id/catalogue-solutions/select when Add Catalogue Solution button is clicked